transport/http/src/main/java/org/kaazing/gateway/transport/http/HttpAcceptProcessor.java

```diff
@@ -361,7 +361,7 @@ public static void setServerHeader(DefaultHttpSession httpSession, HttpResponseM
         if (httpSession != null) {
             ResourceAddress address = httpSession.getLocalAddress();
             boolean serverHeaderEnabled = address.getOption(HttpResourceAddress.SERVER_HEADER_ENABLED);
-            if (serverHeaderEnabled) {
+            if (serverHeaderEnabled && !httpResponse.hasHeader("Server")) {
```

Comment:
The code above appears to set the header if server header is enabled, but do not overwrite.  In addition, when the server header is disabled the Server header is passed through.

Should we have an enhancement to *remove* the passthrough Server header?  (Valid options might be: Enabled, Disabled, Remove, Overwrite)